tests/ami/scripts/fresh_install_rpm.sh.j2

```diff
@@ -28,19 +28,20 @@ cat /etc/*-release
 sudo yum install -y wget
 sudo yum install -y "{{python_package}}"
 
-{% if not package_to_version_is_url -%}
+# Install provided version
+{% if package_to_version_source_type == "install_script" -%}
 wget -O "install-scalyr-agent-2.sh" -q "{{installer_script_url}}"
 sudo bash ./install-scalyr-agent-2.sh --set-api-key "{{scalyr_api_key}}"
-{% endif -%}
+{% elif package_to_version_source_type in ("file", "url") -%}
+    {% if package_to_version_source_type == "url" -%}
+    wget -O to_package.rpm "{{ package_to_version }}"
+    {% endif -%}
 
-# Install latest version
-{% if package_to_version_is_url -%}
-wget -O package.rpm "{{ package_to_version }}"
 sudo rpm -iv package.rpm
 sudo sed -i 's/REPLACE_THIS/{{scalyr_api_key}}/' /etc/scalyr-agent-2/agent.json
-export PACKAGE_TO_VERSION=$(rpm -qp --queryformat '%{VERSION}\n' "package.rpm" | sed "s/_.*//g")
+export PACKAGE_TO_VERSION=$(rpm -qp --queryformat '%{VERSION}\n' "to_package.rpm" | sed "s/_.*//g")
 {% else -%}
-sudo yum install -y "scalyr-agent-2-{{package_to_version}}"
+sudo apt-get install -y "scalyr-agent-2={{package_to_version}}"
```

Comment:
Hm, why the apt-get step here? This script file is used for rpm packages.

Reply:
Just the same reason as above..